_studio/shared/umc/codec/vc1_dec/include/umc_vc1_dec_task_store.h

```diff
@@ -347,9 +341,9 @@ namespace UMC
         uint32_t m_iNumFramesProcessing;
         uint32_t m_iNumDSActiveinQueue;
 
-        vm_mutex m_mDSGuard;
+        std::mutex m_mDSGuard;
 
-        vm_mutex** m_pGuardGet;
+        std::mutex** m_pGuardGet;
```

Comment:
`std::vector<std::unique_ptr<std::mutex> >`
?

Reply:
I guess, remove using m_pSHeap - is good idea, but lets not mix different changes in one PR